### PR TITLE
WIP: virt-manager: update to 2.0.0.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/libxml2-python/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'libxml2-python'
+pkgname=libxml2-python
+version=2.9.8
+revision=1
+wrksrc="${pkgname%-python}-${version}"
+build_wrksrc="python"
```
Comment (maxice8) [inline srcpkgs/libxml2-python/template]: no need to be quoted

**Comment by Cogitri** [comment]
Mind taking a look at this @asergi?

**Comment by Hoshpak** [comment]
Works fine for me. I wonder however why you split the templates. Wasn't it possible to also build the python3 stuff from the libxml2 template? At the very least the revision of the new package should be higher than that of the old one so people don't have to downgrade manually.

CC @thypon as the maintainer of the virt-manager package.

**Comment by Cogitri** [comment]
> I wonder however why you split the templates. Wasn't it possible to also build the python3 stuff from the libxml2 template?

Because that way we can use the `python-module` build style instead of manually building python2&3.

**Comment by Hoshpak** [comment]
Alright, then please add a comment to the libxml2 template to keep it in sync with the python bindings (like in the thunderbird template). libxml2 is not updated that often but we shouldn't risk incompatibilities by shipping different versions of the library and the python bindings.

**Comment by Hoshpak** [comment]
@Cogitri I would be inclined to merge once travis finishes the currently running build and gives the green light. Is that alright with you or do you still want to test or change something?

**Comment by Hoshpak** [comment]
I've tested it with my libvirt VMs just before the last two changes I asked for which didn't change the functionality so I'm sure it works at least for the features I use.

**Comment by Cogitri** [comment]
Alright, great. Thanks for testing

**Pull request merged**